stores/hostdb_test.go

```diff
@@ -462,7 +462,7 @@ func TestRecordScan(t *testing.T) {
 
 	// Record a scan.
 	firstScanTime := time.Now().UTC()
-	subnets := []string{"212.1.96.0/24", "38.135.51.0/24"}
+	subnets := []string{"212.1.96.0", "38.135.51.0"}
```

Comment:
We need some database migration code for existing users in `internal/sql/migrations.go`. 

The easiest way forward is probably to drop the `subnets` column from the database and add a `resolved_addresses` column instead.